socorro/processor/processor_app.py

```diff
@@ -131,6 +142,45 @@ def _transform(self, crash_id):
                 crash_id
             )
             self.config.logger.info('saved - %s', crash_id)
+        except Exception as exception:
+            # Immediately capture this as local variables.
+            # During this error handling we're going to be using other
+            # try:except: constructs (e.g. swallowing raven send errors)
+            # so we can't reference `sys.exc_info()` later.
+            exc_type, exc_value, exc_tb = sys.exc_info()
+
+            if self.config.sentry and self.config.sentry.dsn:
+                try:
+                    if isinstance(exception, PolyStorageError):
```

Comment:
I think you can make this more generic, and not rely on ``PolyStorageError``. Maybe use ``collections.Sequence`` instead? That way you catch most iterables. 

```python
In [9]: isinstance([], collections.Sequence)
Out[9]: True

In [10]: isinstance(Exception('test'), collections.Sequence)
Out[10]: False
```

Note that you can't use ``iter`` because ``Exception`` is an iterable.